src/appmetrics.cpp

```diff
@@ -677,11 +697,13 @@ static bool isGlobalAgent(Local<Object> module) {
 static bool isGlobalAgentAlreadyLoaded(Local<Object> module) {
     Nan::HandleScope scope;
     Local<Object> cache = getRequireCache(module);
-    Local<Array> props = cache->GetOwnPropertyNames();
+    Local<Context> context = Nan::GetCurrentContext();
+    Local<Array> props = cache->GetOwnPropertyNames(context).ToLocalChecked();
```

Comment:
```suggestion
    Local<Array> props = Nan::GetOwnPropertyNames(cache);
```
https://github.com/nodejs/nan/blob/master/doc/maybe_types.md#api_nan_get_own_property_names